easybuild/easyconfigs/g/GffCompare/GffCompare-0.10.1-foss-2016b.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'GffCompare'
+version = '0.10.1'
+git_commit_id = '305b7f80eb33171d39692bc69ea487c1ef9e2275'
```

Comment:
I would only use the commit ID in `sources`, since it's the commit ID for `gclib`, not for `GffCompare`?

Reply:
@boegel Yes, it is the commit ID for `gclib`. I changed this.